src/vs/platform/webview/electron-main/webviewProtocolProvider.ts

```diff
@@ -130,13 +130,12 @@ export class WebviewProtocolProvider extends Disposable {
 			const uri = URI.parse(request.url);
 			const entry = WebviewProtocolProvider.validWebviewFilePaths.get(uri.path);
 			if (typeof entry === 'string') {
-				let url: string;
-				if (uri.path.startsWith('/electron-browser')) {
-					url = require.toUrl(`vs/workbench/contrib/webview/electron-browser/pre/${entry}`);
-				} else {
-					url = require.toUrl(`vs/workbench/contrib/webview/browser/pre/${entry}`);
-				}
-				return callback(decodeURIComponent(url.replace(`${Schemas.file}://`, '')));
+				const relativeResourcePath = uri.path.startsWith('/electron-browser')
```

Comment:
That logic is a bit error prone as soon as e.g. we rename everything to `electron-sandbox`. I would have not thought we have a string based dependency to this like that.

Reply:
The `electron-browser` string actually comes from https://github.com/microsoft/vscode/blob/88929c7a45ec3d1b0c570113cf222ff92a860f60/src/vs/workbench/contrib/webview/electron-browser/webviewElement.ts#L213 so I don't think it should break when we rename things

Let me know if you have any ideas how to avoid hardcoding ``vs/workbench/contrib/webview/electron-browser`. I think it's required here

Reply:
Oh ok, you set the string yourself and it is not bound to the file system structure. I would change it to `native` to make sure we are not confusing it with our layer system.